src/_langs/en/tools/javascript/command-line/events.markdown

```diff
@@ -3,40 +3,93 @@ rss: false
 layout: article
 title: "Monitor Events"
 seotitle: "Monitor Events Using the Chrome DevTools Command Line API"
-description: "TBD description."
-introduction: "TBD introduction."
+description: "The Chrome DevTools Command Line API offers various ways to observe and inspect Event Listeners"
+introduction: "The Chrome DevTools Command Line API offers various ways to observe and inspect Event Listeners"
 article:
   written_on: 2015-04-14
-  updated_on: 2015-04-14
+  updated_on: 2015-05-11
   order: 2
 authors:
   - megginkearney
+  - flaviocopes
 priority: 0
 collection: command-line
 key-takeaways:
-  tldr-tbd:
-    - TBD tldr.
-remember:
-  note-tbd:
-    - TBD note.
+  tldr:
+    - Use monitorEvents() to listen to events of a certain type
+    - Use unmonitorEvents() to stop listening
+    - Use getEventListeners() to get the event listeners of a DOM element
+    - Use the Event Listeners Inspector panel to get information on event listeners
 ---
 {% wrap content %}
 
-TBD. Specified parts of this doc: https://developer.chrome.com/devtools/docs/console
 
-TBD. Includes:
+# Monitor Events
 
-** Monitoring events
-** View event listeners registered on objects
+JavaScript plays a central role in interactive pages, and the Browser can provides you some useful tools to debug events and event handlers.
 
-{% include modules/takeaway.liquid list=page.key-takeaways.tldr-tbd %}
+## Monitoring events
+
+The [monitorEvents()](https://developer.chrome.com/devtools/docs/commandline-api#monitoreventsobject-events) method instructs the DevTools to log information on the specified targets.
+
+The first parameter is the object to monitor.
+All events return if the second parameter is not provided.
+To specify the events to listen to you may pass either a string or an array of strings as the second parameter.
+
+
+Listen to click events on the body of the page.
+
+    monitorEvents(document.body, "click");
+
+If a supported *event type* that the DevTools maps to a set of standard event names then it will listen to the events for that type.
+The [Command Line API](https://developer.chrome.com/devtools/docs/commandline-api#monitoreventsobject-events) has a full mapping of *event types* to the events they cover.
+
+To stop monitoring events you call the `unmonitorEvents()` method and give it the object to stop monitoring.
+
+Stop listening to events on the `body` object
+
+    unmonitorEvents(document.body);
```

Comment:
I moved it on its own code block as a `code example` instead of putting it inline. Does it make sense? 

![screen shot 2015-05-12 at 08 38 42](https://cloud.githubusercontent.com/assets/11783388/7581356/58fdeae4-f882-11e4-9487-1846bd96368a.png)

vs

![screen shot 2015-05-12 at 08 36 59](https://cloud.githubusercontent.com/assets/11783388/7581342/2f677484-f882-11e4-978f-85a2c09948b8.png)